so_box_biogeo/diags/estNsq.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+from MITgcmutils import rdmds, densjmd95
+from hspython import loadgrid
+import matplotlib.pyplot as plt
+"""
+    Estimating Brunt-Vaisala frequency using T and S
+    In case when N$^2$ is needed but do not have 'DRHODR' saved, 
+    one can still estimate it using T and S
```

Comment:
Is it worth adding "and the appropriate equation of state" to the end of this docstring?